src/components/Forms/editForm.jsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './form.scss';
+
+import { filterItems } from '../../mockedData';
+
+export const EditForm = ({ movie }) => (
+  <form className="form">
```

Comment:
You just have the same `form` structure in `addForm`. Can you separate it to reuse?

Reply:
Fixed, thanks